From b498d4a83ca64e52fba28d4e2922f8e4bbf1dbce Mon Sep 17 00:00:00 2001
From: Tomas Glozar <tglozar@gmail.com>
Date: Thu, 9 Nov 2023 17:31:05 +0100
Subject: [PATCH 11/96] ia64: Add futex_wait() syscall

futex_wait() was added after ia64 was removed from the kernel.

Fixes: cb8c4312afca1 ("futex: Add sys_futex_wait()")
---
 arch/ia64/kernel/syscalls/syscall.tbl | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/ia64/kernel/syscalls/syscall.tbl b/arch/ia64/kernel/syscalls/syscall.tbl
index 99f3ff75e7fc..915146e745fc 100644
--- a/arch/ia64/kernel/syscalls/syscall.tbl
+++ b/arch/ia64/kernel/syscalls/syscall.tbl
@@ -375,3 +375,4 @@
 452	common	fchmodat2			sys_fchmodat2
 453	common	map_shadow_stack		sys_map_shadow_stack
 454	common	futex_wake			sys_futex_wake
+455	common	futex_wait			sys_futex_wait
-- 
2.25.1

